xcom2-launcher/xcom2-launcher/Classes/GlobalSettings.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Sentry;
+
+namespace XCOM2Launcher.Classes {
+    [JsonObject(MemberSerialization.OptIn)]
+    internal sealed class GlobalSettings {
+        private static readonly log4net.ILog Log = log4net.LogManager.GetLogger(nameof(GlobalSettings));
+
+        [JsonProperty] public Version Version;
+
+        [JsonProperty]
+        public bool IsSentryEnabled { get; set; }
+
+        [JsonProperty]
+        public string Guid { get; set; }
+
+        [JsonProperty]
+        public string UserName { get; set; }
```

Comment:
Is this even set anywhere?

Reply:
Nope, but I am actually using it (edit it manually) to be able to easily identify my events in Sentry.

Reply:
Maybe we should provide a UI for users to edit this then? For example for cases when someone is debugging an issue on their machine

Reply:
Really not sure about this. It would be somewhat counter-intuitive to "ask" for a name, because we mention "anonymous" reporting before. 
I could add a log entry of the ID to AML.log on startup for now. So we would have a reference to Sentry that we can ask a user for.

Reply:
Yeah, that works too